### DISCUSSION TITLE
Newsletter for modular

### DISCUSSION THREAD

**Discussion by Eprahim-taha** [modular/modular#1635] [General]
I am interested in knowing the news of the modular company. I subscribed to the newsletter. I was supposed to receive the newsletter every week in the inbox of my Google account, but I do not receive anything. I do not know what the problem is. The last thing I received was Modverse Weekly - Issue 17.

**Comment by Brian-M-J** [comment]
> The last thing I received was Modverse Weekly - Issue 17.

Week 17 is the latest issue. You can check what the latest issue is [here](https://www.modular.com/newsletters).

**Reply by Eprahim-taha** [reply]
Thanks for the clarification, but why didn't they publish it week 18?

**Reply by Brian-M-J** [reply]
I don't work at Modular so I have no idea.

**Comment by iamtimdavis** [comment]
Thanks - we're working hard on scaling out our team, and we'll get regular newsletter updates back in your inbox shortly! :)

**Reply by Eprahim-taha** [reply]
Thanks. Is there any intention to release Mojo 🔥 0.7 this month?
